domains/client/domain-operator/src/aux_schema.rs

```diff
@@ -159,10 +172,20 @@ where
     Block: BlockT,
     CBlock: BlockT,
 {
-    load_decode(
+    let maybe_raw_value = get_value_raw(
         backend,
         execution_receipt_key(consensus_block_hash).as_slice(),
-    )
+    )?;
+    match maybe_raw_value {
+        None => Ok(None),
+        Some(t) => match ExecutionReceiptFor::<Block, CBlock>::decode(&mut t.as_slice()) {
+            Ok(receipt) => Ok(Some(receipt)),
+            Err(_) => ExecutionReceiptV0For::<Block, CBlock>::decode(&mut t.as_slice())
```

Comment:
Code like this seems to be duplicated in a few places. Is it possible to centralise it, or is that tricky due to generics?

(It's ok to repeat it now, but I'm just imagining in the future where we have to find all similar code and update it with more versions.)